.circleci/config.yml

```diff
@@ -175,11 +200,27 @@ workflows:
   version: 2
   workflow:
     jobs:
-      - build_debug
-      - test_unit
-      - test_instrumented:
+#     This job will run on every commit on any branch
+      - build_debug:
+          filters:
```

Comment:
Since the regex allows all branches so nothing is filtered out. So is the `filters` section required?